src/huggingface/forte/huggingface/token_classification.py

```diff
@@ -0,0 +1,221 @@
+    specified entry type in the input pack and the prediction result goes to the
+    user specified attribute name of that entry type in the output pack. User
+    could provide task and models in the config to perform NER, POS,
+    word segmentation task, etc.
+
+    """
+
+    def __init__(self):
+        super().__init__()
+        self.classifier = None
+
+    def set_up(self):
+        self.classifier = pipeline(
+            "ner",
+            model=self.configs.model_name,
+            tokenizer=self.configs.tokenizer,
+            framework=self.configs.framework,
+            device=self.configs.cuda_device,
+        )
+
+    def initialize(self, resources: Resources, configs: Config):
+        super().initialize(resources, configs)
+        self.set_up()
+
+    def _process(self, input_pack: DataPack):
+        """Perform HuggingFace NER Pipeline on the input data pack.
+
+        Args:
+            input_pack: Input pack to fill
+        Returns:
+        """
+        input_path, input_module = self.configs.entry_type.rsplit(".", 1)
+        mod = importlib.import_module(input_path)
+        input_entry = getattr(mod, input_module)
+
+        output_path, output_module = self.configs.output_entry_type.rsplit(
+            ".", 1
+        )
+        mod = importlib.import_module(output_path)
+        output_entry = getattr(mod, output_module)
+
+        for entry_specified in input_pack.get(entry_type=input_entry):
+            result = self.classifier(entry_specified.text)
+
+            if self.configs.strategy == "bio-merge":  # Merge BIO tagging
+                result_types, result_indices = self._merge_bio_tokens(result)
+
+            else:
+                result_indices = []
+                result_types = []
+                for token in result:
+                    start, end = token["start"], token["end"]
+                    result_types.append(token["entity"])
+                    result_indices.append((start, end))
+
+            for type, (start, end) in zip(result_types, result_indices):
+                entity = output_entry(
+                    pack=input_pack,
+                    begin=entry_specified.span.begin + int(start),
+                    end=entry_specified.span.begin + int(end),
+                )
+
+                if self.configs.task == "ner":
+                    if output_module == "EntityMention":
+                        entity.ner_type = type
+                    else:
+                        try:
+                            entity.ner = type
+                        except KeyError as exc:
+                            raise ProcessorConfigError(
+                                f"NER type was not stored in the given "
+                                f"'output_entry_type': "
+                                f"{self.configs.output_entry_type}."
+                                f"EntityMention or Token was recommended."
+                            ) from exc
```

Comment:
We can support any entry type, so please don't hard code `EntityMention` and `Token` here.